s/frontend/components/common/five-stars/namaka-review.ts

```diff
@@ -0,0 +1,26 @@
+import {renderFiveStarRating, stateBuddy} from './render-five-star-display.js';
```

Comment:
our javascript doesn't use semicolons, please remove them all

Reply:
yess i need to get used to that